### PR TITLE
React quickstart: Logout from Auth0 authorization server

### PR DESCRIPTION
**Pull request by alexisluque**
Clear session at Auth0 authorization server calling `/v2/logout` endpoint when perform logout.
Sample also updated at: auth0-samples/auth0-react-samples#108.

### CODE REVIEW

File: articles/quickstart/spa/react/01-login.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ github:
 contentType: tutorial
 useCase: quickstart
 ---
-<%= include('../_includes/_getting_started', { library: 'React', callback: 'http://localhost:3000/callback' }) %>
+<%= include('../_includes/_getting_started', { library: 'React', callback: 'http://localhost:3000/callback', showLogoutInfo: true, showWebOriginInfo: true }) %>
```
Comment (lbalmaceda) [inline articles/quickstart/spa/react/01-login.md:15]: @alexisluque what is this new line doing?

Comment (alexisluque) [inline articles/quickstart/spa/react/01-login.md:15]: @lbalmaceda assigning `true` to `showLogoutInfo` and `showWebOriginInfo` variables shows [logout URL](https://github.com/auth0/docs/blob/master/articles/_includes/_logout_url.md) and [allowed web origins](https://github.com/auth0/docs/blob/master/articles/_includes/_web_origins.md) snippets in 
 [getting started](https://github.com/auth0/docs/blob/master/articles/quickstart/spa/_includes/_getting_started.md) snippet, as is done in [VueJS QS](https://auth0.com/docs/quickstart/spa/vuejs#configure-logout-urls).

Comment (lbalmaceda) [reply articles/quickstart/spa/react/01-login.md:15]: I wasn't aware of this dynamic values.. Thanks for clarifying that 👍

**Review by lbalmaceda — APPROVED** [review]

**Comment by albertoperdomo** [comment]
>@albertoperdomo albertoperdomo requested a review from lbalmaceda just now

This was accidental, sorry @lbalmaceda.

**Pull request merged**